cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -605,43 +615,35 @@ define(["jquery", "edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers", "common/j
             });
 
             describe("Subsection", function() {
-                var getDisplayNameWrapper, setEditModalValues, mockServerValuesJson,
-                    selectDisableSpecialExams, selectBasicSettings, selectAdvancedSettings,
-                    selectAccessSettings, selectTimedExam, selectProctoredExam, selectPracticeExam,
+                var getDisplayNameWrapper, setEditModalValues, setContentVisibility,  mockServerValuesJson,
+                    selectDisableSpecialExams, selectTimedExam, selectProctoredExam, selectPracticeExam,
                     selectPrerequisite, selectLastPrerequisiteSubsection, checkOptionFieldVisibility;
 
                 getDisplayNameWrapper = function() {
                     return getItemHeaders('subsection').find('.wrapper-xblock-field');
                 };
 
-                setEditModalValues = function (start_date, due_date, grading_type, is_locked) {
+                setEditModalValues = function (start_date, due_date, grading_type) {
                     $("#start_date").val(start_date);
                     $("#due_date").val(due_date);
                     $("#grading_type").val(grading_type);
-                    $("#staff_lock").prop('checked', is_locked);
-                };
-
-                selectDisableSpecialExams = function() {
-                    this.$("input.no_special_exam").prop('checked', true).trigger('change');
-                };
-
-                selectBasicSettings = function() {
-                   this.$(".modal-section .settings-tab-button[data-tab='basic']").click();
                 };
 
-                selectAdvancedSettings = function() {
-                   this.$(".modal-section .settings-tab-button[data-tab='advanced']").click();
+                setContentVisibility = function (visibility) {
+                    $('input[name=content-visibility][value='+visibility+']').prop('checked', true);
                 };
 
-                selectAccessSettings = function() {
-                   this.$(".modal-section .settings-tab-button[data-tab='access']").click();
+                selectDisableSpecialExams = function() {
+                    this.$("input.no_special_exam").prop('checked', true).trigger('change');
                 };
 
                 selectTimedExam = function(time_limit, hide_after_due) {
                     this.$("input.timed_exam").prop('checked', true).trigger('change');
                     this.$(".field-time-limit input").val(time_limit);
                     this.$(".field-time-limit input").trigger('focusout');
-                    this.$('.field-hide-after-due input').prop('checked', hide_after_due).trigger('change');
+                    if (hide_after_due) {
```

Comment:
To follow up here:
- I've traced the code locally, and setting `hide_after_due` in the new fashion still sets the value in SQL for the timed exam, so we're all set there.
- For the test method specifically, the tests that are calling this are just verifying that selected options, when set, are returned from the modal on save. Thus, I've elected to always set `hide_after_due` visibility in `selectTimedExam`.

This comment will be lost in the diff after I push my latest commit, but I wanted to keep the conversation in context.